test/jdk/java/nio/channels/Channels/ReadXBytes.java

```diff
@@ -23,22 +23,25 @@
 
 /*
  * @test
- * @bug 8268435
+ * @bug 8268435 8274780
  * @summary Verify ChannelInputStream methods readAllBytes and readNBytes
  * @library ..
  * @library /test/lib
  * @build jdk.test.lib.RandomFactory
  * @modules java.base/jdk.internal.util
- * @run testng/othervm -Xmx8G ReadXBytes
+ * @run testng/othervm/timeout=3600 -Xmx8G ReadXBytes
```

Comment:
I'm invoking jtreg directly, following [Running tests using jtreg](http://openjdk.java.net/jtreg/runtests.html). That page doesn't mention `-timeoutFactor`.

Reply:
> I'm invoking jtreg directly, following [Running tests using jtreg](http://openjdk.java.net/jtreg/runtests.html). That page doesn't mention `-timeoutFactor`.

Okay, I think RunTests.gmk sets it to 4 but it can be overridden.

Reply:
Should I lower the timeout to 900 then? With a factor of 4, that should be enough to run the test even on fairly slow machines.